web/components/issues/issue-detail/root.tsx

```diff
@@ -99,14 +99,22 @@ export const IssueDetailRoot: FC<TIssueDetailRoot> = observer((props) => {
         showToast: boolean = true
       ) => {
         try {
-          await updateIssue(workspaceSlug, projectId, issueId, data);
+          const issueUpdatePromise = updateIssue(workspaceSlug, projectId, issueId, data);
           if (showToast) {
-            setToastAlert({
-              title: "Issue updated successfully",
-              type: "success",
-              message: "Issue updated successfully",
+            setPromiseToast(issueUpdatePromise, {
```

Comment:
I feel like issue update doesn't even need a toast, please check with Dilip and remove it completely